maudtools/plotting.py

```diff
@@ -3,8 +3,8 @@
 import arviz as az
```

Comment:
UPDATE: still works fine. The problem now is that retrieving yconc and fluxes is more complicated since it is not shaped into a `experiment, metabolites` array, but that's an issue with Maud itself (using `maud-metabolic-models==0.4.0.2`).